### DISCUSSION TITLE
数据源插件化之后，可以支持Kafka，HBase，Redis等不能通过JDBC连接的数据源吗？

### DISCUSSION THREAD

**Discussion by LiuBodong** [apache/dolphinscheduler#6254] [General]
当前版本，所有数据源都是通过`JDBC`连接的，目前社区正在规划将数据源进行插件化，我想知道，当数据源插件化完成之后，能够通过编写插件的方式连接到`Kafka`，`HBase`等其他数据源吗？

**Comment by Narcasserun** [comment]
JDBC can be supported. HBase and Kafka can be extended by themselves

**Reply by Narcasserun** [reply]
#6071
